actions/loadconfig.yaml

```diff
@@ -28,17 +28,25 @@ parameters:
     config_file:
         type: "string"
         description: "Absolute path to configuration file on local system that will be sent to the device"
-        required: true
+        required: false
         position: 4
+    config_text:
+        type: "string"
+        description: "A command that will be sent to the device"
```

Comment:
- Mention that `config_file` takes precedence if both given
- It's not a command actually, it's configuration in text format.

so, `"A configuration text that will be sent to the device. Need to specify either config_file or config_text. If both given, config_file takes precedence."`